.storybook/data/migrated-components.json

```diff
@@ -0,0 +1,317 @@
+    },
+    {
+      "name": "contextualhelp",
+      "title": "Contextual help",
+      "url": "contextual-help"
+    },
+    {
+      "name": "dial",
+      "title": "Dial",
+      "url": "dial"
+    },
+    {
+      "name": "dialog",
+      "title": "Dialog",
+      "url": "dialog"
+    },
+    {
+      "name": "divider",
+      "title": "Divider",
+      "url": "divider"
+    },
+    {
+      "name": "dropzone",
+      "title": "Drop zone",
+      "url": "drop-zone"
+    },
+    {
+      "name": "fieldgroup",
+      "title": "Field group",
+      "url": "field-group"
+    },
+    {
+      "name": "fieldlabel",
+      "title": "Field label",
+      "url": "field-label"
+    },
+    {
+      "name": "form",
+      "title": "Form",
+      "url": "form"
+    },
+    {
+      "name": "helptext",
+      "title": "Help text",
+      "url": "help-text"
+    },
+    {
+      "name": "icon",
+      "title": "Icon",
+      "url": "icon"
+    },
+    {
+      "name": "illustratedmessage",
+      "title": "Illustrated message",
+      "url": "illustrated-message"
+    },
+    {
+      "name": "infieldbutton",
+      "title": "In-field button",
+      "url": "in-field-button"
+    },
+    {
+      "name": "infieldprogresscircle",
+      "title": "In-field progress circle",
+      "url": "in-field-progress-circle"
+    },
+    {
+      "name": "inlinealert",
+      "title": "In-line alert",
+      "url": "in-line-alert"
+    },
+    {
+      "name": "link",
+      "title": "Link",
+      "url": "link"
+    },
+    {
+      "name": "menu",
+      "title": "Menu",
+      "url": "menu"
+    },
+    {
+      "name": "meter",
+      "title": "Meter",
+      "url": "meter"
+    },
+    {
+      "name": "opacitycheckerboard",
+      "title": "Opacity checkerboard",
+      "url": "opacity-checkerboard"
+    },
+    {
+      "name": "pagination",
+      "title": "Pagination",
+      "url": "pagination"
+    },
+    {
+      "name": "picker",
+      "title": "Picker",
+      "url": "picker"
+    },
+    {
+      "name": "pickerbutton",
+      "title": "Picker button",
+      "url": "picker-button"
+    },
+    {
+      "name": "popover",
+      "title": "Popover",
+      "url": "popover"
+    },
+    {
+      "name": "progressbar",
+      "title": "Progress bar",
+      "url": "progress-bar"
+    },
+    {
+      "name": "progresscircle",
+      "title": "Progress circle",
+      "url": "progress-circle"
+    },
+    {
+      "name": "radio",
+      "title": "Radio",
+      "url": "radio"
+    },
+    {
+      "name": "rating",
+      "title": "Rating",
+      "url": "rating"
+    },
+    {
+      "name": "search",
+      "title": "Search field",
+      "url": "search-field"
+    },
+    {
+      "name": "slider",
+      "title": "Slider",
+      "url": "slider"
+    },
+    {
+      "name": "statuslight",
+      "title": "Status light",
+      "url": "status-light"
+    },
+    {
+      "name": "stepper",
+      "title": "Number field",
+      "url": "number-field"
```

Comment:
Does this one show up out of alphabetical order for you in the list?

<img width="128" height="175" alt="image" src="https://github.com/user-attachments/assets/7011ce26-14d8-4d2f-8332-93c421b3f31b" />

Not a deal breaker at all, but worth mentioning.

Reply:
It's probably because only that component's `title` in Storybook is Number field- it's package and component name that we ship are still Stepper.

Reply:
lemme know what you think of the sorting functions! 

<img width="183" height="152" alt="Screenshot 2025-08-15 at 5 18 00 PM" src="https://github.com/user-attachments/assets/6d646be7-eac1-47a0-9eb8-df92875b27a3" />